Formula/mprocs.rb

```diff
@@ -0,0 +1,18 @@
+class Mprocs < Formula
+  desc "Run multiple commands in parallel"
+  homepage "https://github.com/pvolok/mprocs"
+  url "https://github.com/pvolok/mprocs/archive/refs/tags/v0.6.0.tar.gz"
+  sha256 "c9314e81b0a5584e73a1afc2c1b5b83df0ed31213992447324cb95abbda618ce"
+  license "MIT"
+  head "https://github.com/pvolok/mprocs.git", branch: "master"
+
+  depends_on "rust" => :build
+
+  def install
+    system "cargo", "install", *std_cargo_args
+  end
+
+  test do
+    assert_match version.to_s, shell_output("#{bin}/mprocs --version")
```

Comment:
Can you propose a more substantial test than checking the version or calling for help? These tests rarely ever fail even for broken installations.

>We want tests that don’t require any user input and test the basic functionality of the application. For example `foo build-foo input.foo` is a good test and (despite their widespread use) `foo --version` and `foo --help` are bad tests. 

Ref. https://docs.brew.sh/Formula-Cookbook#add-a-test-to-the-formula

Reply:
Mprocs is a TUI app. Writing a meaningful test would require spawning mprocs and then passing input to stdin with timeouts. I saw another tui app vifm only checking `--version`. Do I have to come up with some kind of tui interaction emulation?

Reply:
I added a test that makes mprocs to run `echo hello mprocs` and makes sure it show the output.